docs/UserGuide.md

```diff
@@ -54,6 +54,86 @@ These are symbols we will be using to highlight certain information to you.
 | :bulb:               | Tips to optimise the use of Foster Family |
 | :exclamation:        | Warning about the usage of commands       |
 
+--------------------------------------------------------------------------------------------------------------------
+## User Interface
+There are mainly two different screens that you may face using Foster Family. 
+
+### The Main View 
+![Ui](images/Ui.png)
+This is the main view that welcomes you when you first open up Foster Family.
```

Comment:
can you call this main window instead to standardise

Reply:
Ok! Thanks!